util/helm/cmd.go

```diff
@@ -44,7 +44,11 @@ func (c Cmd) run(args ...string) (string, error) {
 	cmd := exec.Command(c.binaryName, args...)
 	cmd.Dir = c.WorkDir
 	cmd.Env = os.Environ()
-	cmd.Env = append(cmd.Env, fmt.Sprintf("HELM_HOME=%s", c.helmHome))
+	cmd.Env = append(cmd.Env,
+		fmt.Sprintf("XDG_CACHE_HOME=%s", c.helmHome),
```

Comment:
I'm not sure whether we should point `XDG_CACHE_HOME` to same location as `HELM_HOME` - how about `$HELM_HOME/cache`? Also, I found a discussion around this at https://github.com/helm/helm/issues/7206, but I'm not sure if this will apply to us (except when users have a sidecar that installs Helm plugins).

Reply:
Thanks for pointing this out. I think it is better to use `$HELM_HOME/cache` at least for consistency. Updated the `XDG_CACHE_HOME` env variable.